build.ps1

```diff
@@ -69,6 +69,9 @@ bazel build //... `
 bazel shutdown
 
 if ($env:SKIP_TESTS -ceq "False") {
+    # Generate mapping from shortened scala-test names on Windows to long names on Linux and MacOS.
+    ./ci/remap-scala-test-short-names.ps1 >scala-test-suite-name-map.json
+
```

Comment:
Not quite sure where the issue comes from, but, at least as of the last build resuls I can find, this seems to be producing a UTF-16 file, which is technically not valid JSON. I solved it locally with `iconv`.